packages/networks/src/networks/testnets/zksync-sepolia.ts

```diff
@@ -11,6 +11,7 @@ export const zkSyncSepolia: ChildNetworkConfig = {
         l1Messenger: '0x681A1AFdC2e06776816386500D2D461a6C96cB45',
       },
       parent: {
+        nativeTokenVault: '0x746DBBa1edfBe1b547c87189eFE91B77d53d9E39',
```

Comment:
wrong address it seems https://sepolia-era.zksync.network/address/0x746DBBa1edfBe1b547c87189eFE91B77d53d9E39